GenderPayGap.WebUI/Models/AddOrganisation/AddOrganisationManualViewModel.cs

```diff
@@ -31,18 +34,44 @@ public class AddOrganisationManualViewModel : GovUkViewModel
 
         public string Query { get; set; }
 
+        [GovUkValidateCharacterCount(MaxCharacters = 100)]
+        [GovUkDisplayNameForErrors(NameAtStartOfSentence = "Employer name", NameWithinSentence = "employer name")]
         [GovUkValidateRequired(ErrorMessageIfMissing = "Enter the name of the employer")]
         public string OrganisationName { get; set; }
 
+        [GovUkDisplayNameForErrors(NameAtStartOfSentence = "PO Box", NameWithinSentence = "PO Box")]

```

Comment:
Should the b be lower case for consistency (i.e. PO box, instead of PO Box). I guess this is an english question? 🤔

Reply:
hmm I _think_ it should be PO Box 
> A post office box, or commonly known as a PO Box or a Postal Box, is a lockable mailbox located in a post office station

Reply:
I admit my defeat! 😅